pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: html
-version: 0.13.0
+version: 0.14.0
```

Comment:
Fine to do a +1 release here – no other breaking changes. Folks won't be able to get this without bumping their SDK – let's avoid a cascading dependency dance.